spec/plugin_mixin/aws_config_spec.rb

```diff
@@ -78,3 +86,90 @@ def aws_service_endpoint(region)
     end
   end
 end
+
+describe LogStash::PluginMixins::AwsConfig::V2 do
+  let(:settings) { {} }
+
+  subject { DummyInputAwsConfigV2.new(settings).aws_options_hash }
+
+  describe 'config credential' do
+    subject { DummyInputAwsConfigV2.new(settings).aws_options_hash[:credentials] }
+
+    context 'in credential file' do
```

Comment:
Well, it doesn't return the same structure?
- V1, return a hash.
- V2, return a hash and a credential instance.